src/App.jsx

```diff
@@ -14,33 +17,103 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_APLHABET = 'alphabet';
+const SORT_LENGTH = 'length';
+const INITIAL_STATE = '';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const prepareGoods = (goods, action, reverse) => {
+  const preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (action) {
+    switch (action) {
+      case SORT_APLHABET:
+        preparedGoods.sort();
+        break;
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+      case SORT_LENGTH:
+        preparedGoods.sort((good1, good2) => good1.length - good2.length);
+        break;
+
+      default:
+        break;
+    }
+  }
+
+  if (reverse) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+};
+
+const RenderGoodList = ({ goods }) => {
```

Comment:
The component RenderGoodList is interacting with the DOM directly, which is against the guidelines. It would be better to use React as much as possible. You can use the map function inside the return statement of the component.